test/unit/test-amp-story-player.js

```diff
@@ -370,5 +370,80 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
         'Story URL not found in the player: https://example.com/story6.html'
       );
     });
+
+    it('back button should be created', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      playerEl.setAttribute('button', 'back');
+      appendStoriesToPlayer(playerEl, 5);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+
+      expect(playerEl.shadowRoot.querySelector('a')).to.exist;
```

Comment:
Can we be a bit more specific with the `querySelector` to make sure it's the button we want?

How about `a.amp-story-player-back-button` ?